authors/apps/articles/tests/test_rate_article.py

```diff
@@ -0,0 +1,81 @@
+from django.urls import reverse
+from rest_framework import status
+from ...authentication.tests import base_class
+from .test_data import test_rate_article_data
+from ..models import Rating, Article
+
+
+class TestRatingModel(base_class.BaseTest):
+    """
+    This class tests the Rating View
+    """
+
+    def setUp(self):
+        super().setUp()
+        self.user_author = self.activated_user()
+        self.client.force_authenticate(user=self.user_author)
+        self.article = self.create_article(self.user_author)
+        self.url_rate_article = reverse('articles:article-rates',
+                                        kwargs={"slug": self.article.slug})
+        self.url_articles = reverse('articles:articles')
+
+    """unit tests for module under rating articles"""
```

Comment:
@missvicki I don't understand what this docstring is doing here

Reply:
I had added it to indicate that this is where unit tests start. do you think i should make it a comment.

Reply:
@missvicki ohh.. sorry my mistake